src/Helpers/password.js

```diff
@@ -0,0 +1,151 @@
+const validationOptions = {
+  lowercase: '[a-z]',
+  uppercase: '[A-Z]',
+  symbol: '[^a-zA-Z0-9]',
+  number: '[0-9]'
+}
+
+export const Validation = (validation) => {
+  const {
+    HTMLValidate = 0,
+    required = 0,
+    min = 0,
+    max = 0,
+    email = 0,
+    phone = 0,
+    url = 0,
+    ...restValidations
+  } = validation
```

Comment:
Lets change this code . Take only new added validation options from main validation object (regEx also cause if it exists we have make it be array  ) ,  then iterate on taken objects and then return final validation object .